src/main/resources/checkstyle_modules.json

```diff
@@ -0,0 +1,2340 @@
+[
+  {
+    "name": "Checker",
```

Comment:
@Luolc See https://github.com/checkstyle/regression-tool/issues/37 .
Lets remove this file from `main` and use it only in test for our UTs.
It will only be used in #37 .